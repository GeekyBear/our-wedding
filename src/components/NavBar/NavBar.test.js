import React from "react";
import { render, cleanup } from "@testing-library/react";
import NavBar from "./NavBar";

afterEach(cleanup);

it("should take a snapshot", () => {
  const { asFragment } = render(<NavBar />);
  expect(asFragment(<NavBar />)).toMatchSnapshot();
});
