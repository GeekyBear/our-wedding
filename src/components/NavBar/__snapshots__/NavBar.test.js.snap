// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <h1>
    NavBar
  </h1>
</DocumentFragment>
`;
